apps/greenhouse-management/src/components/Auth.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react"
+import { Spinner, Stack } from "juno-ui-components"
+import { useAuthAppLoaded, useIsLoggedIn } from "./StoreProvider"
+import { Transition } from "@tailwindui/react"
+
+// Adds a loading screen while during auth
+// Shows children when auth is complete
+
+const Auth = ({ children }) => {
+  const authAppLoaded = useAuthAppLoaded()
```

Comment:
I don't see where you use this var. This was just used to display the second loading icon with the sap logo.